dev-environment.yml

```diff
@@ -4,14 +4,16 @@ channels:
 dependencies:
   - python>=3.8
   - proj>=7.2
+  - geopandas>=0.10.0
   - fiona
   - shapely
   - numba
   - numpy
   - matplotlib
   - opencv
+  - openh264=2.3.0
```

Comment:
to be removed later when the issue is fixed, no?

Reply:
Added an issue so we remove it in the future.

Reply:
:+1: